### PR TITLE
Bring the website back to life

### PR DESCRIPTION
**Pull request by danielgoodwin97**
No problem brother. I got you. :fist_raised:

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,188 +1,149 @@
...
+
+const express = require('express');
+const app = express();
+const http = require('http').Server(app);
+const io = require('socket.io')(http);
+const fs = require('fs');
+const _ = require('./public/underscore.js')._;
+
+fs.readFile('public/common.js', 'utf8', (err, data) => {
+    if (err) throw err;
```
Comment (robbie-thompson) [inline server.js]: Let's not use short-hand functions.
```suggestion
    if (err) {
        throw err;
    }
```

File: server.js [hunk 2]
```diff
@@ -1,188 +1,149 @@
...
-// Create all the entities in the game (initially a bunch of bots)
-var entities = [];
+app.get('/', (req, res) => {
+    res.sendFile(__dirname + '/index.html');
+});
 
-// Enter gameloop
-gameLoop();
+fs.readFile(__dirname + '/public/map.json', 'utf8', (err, data) => {
+    if (err) throw err;
```
Comment (robbie-thompson) [inline server.js]:
```suggestion
    if (err) {
        throw err;
    }
```

File: server.js [hunk 3]
```diff
@@ -1,188 +1,149 @@
...
-            player.orientation = 'right';
-        } else if (data == '') {
-            player.action = 'stationary';
-        }
+    socket.on('direction-update', (data) => {
+        player.action = data ? 'walk' : 'stationary';
+        player.orientation = data === 'N' ? 'up' :
+            data === 'S' ? 'down' :
+                data === 'W' ? 'left' :
+                    data === 'E' ? 'right' : player.orientation;
```
Comment (robbie-thompson) [inline server.js]: Let's get this in the same format you've used elsewhere.
```suggestion
        switch (data) {	
            case 'N': player.orientation = 'up';    break;
            case 'S': player.orientation = 'down';  break;
            case 'W': player.orientation = 'left';  break;
            case 'E': player.orientation = 'right'; break;
            default: break; // Keep existing orientation if none of the cases match
        }
```

**Review by robbie-thompson — CHANGES_REQUESTED** [review]
LGTFM.

File: server.js (hunk 2 above)
Comment (thedanielgoodwin) [reply server.js]: Done.

File: server.js (hunk 1 above)
Comment (thedanielgoodwin) [reply server.js]: Done.

File: server.js (hunk 3 above)
Comment (thedanielgoodwin) [reply server.js]: Done.

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Review by robbie-thompson — APPROVED** [review]

**Comment by danielgoodwin97** [comment]
@shawesome Looks like it's got the go ahead from @robbie-thompson

**Review by martinshawruroc — APPROVED** [review]

**Pull request merged**